pkg/tf2pulumi/convert/testdata/partial_attribute_lookup/experimental_pcl/main.pp

```diff
@@ -2,3 +2,16 @@
   inputOne = exampledataunknownDataSource.attr
   inputTwo = exampleunknownResourceType.list[0]
 }
+resource "anotherExample" "simple:index:resource" {
+  inputOne = resourceName.someAttribute
+  inputTwo = someConfig.testAttribute
+}
+output "testUnknownAlreadyDeclaredDataSource" {
+  value = exampledataunknownDataSource.anotherTestAttribute
+}
+output "testUnknownLocalVariable" {
+  value = someVariableName
+}
+output "testUnknownConflictingLocalVariable" {
+  value = theirexample.anotherTestAttribute
```

Comment:
huh nice catch here. This should be `theirExample` I think that's a bug in the uniqifier when adding prefixes. Could you fix that up?